install.bat

```diff
@@ -0,0 +1,35 @@
+@echo off
+
+:: change current directory to the one where this script is located
+:: if executed as administrator it would change to system32 otherwise
+pushd %~dp0
+
+python --version 2>NUL
+if errorlevel 1 goto errorNoPython
+
+if exist %UserProfile%\.platformio\penv\Scripts\ (
```

Comment:
Can we not check if `pio` is present in the path? Is this the best way to check for it?